### ISSUE TITLE
Quotes not properly handled

### ISSUE THREAD

**Issue by michaeldotknox** [fclp/fluent-command-line-parser#74]
It appears that quotes are not properly handled with the arguments.  For instance, if one of the arguments is a path such as "c:\directory to my files", which needs to be in quotes, then the parser will not assign a value.  I'm using the Generic Fluent Command Line Parser.

I can confirm that if I manually strip out the quotes in the beginning and in the end that properties are assigned correctly.  I started to go through the code to find the source of the problem, but I am on a bit of a tight deadline to get out some command line tools.  Once I get past the deadline, I'll take a look and see if I can spot the problem.

Thanks for making a nice tool.

**Comment by QuangT123** [comment]
I had this issue when I was copying parameter values with quotes from emails.
When I manually typed it in it worked - I assume outlooks quote is different

**Comment by michaeldotknox** [comment]
No, I never found a solution to the problem.  I ended up writing my own component to do what I needed to do.

**Comment by siywilliams** [comment]
Hi @QuangT123 & @michaeldotknox 

You've most likely lost interest now but I can't seem to reproduce your issue. If I setup the following and call it using the following cmd:

`app.exe --file-path "c:\directory to my files"`

``` c#
class Args
{
    public string FilePath { get; set; }
}

static void Main(string[] args)
{
    var fclp = new FluentCommandLineParser<Args>();

    fclp.Setup(arg => arg.FilePath)
        .As("file-path");

    var result = fclp.Parse(args);

    // fclp.Object.FilePath is now "c:\directory to my files"
}
```

Was you app a console application, were you using the `string[] args`?

**Comment by elbehery95** [comment]
@siywilliams the issue happens when using "/" instead of "\\"
